release_images.yml

```diff
@@ -4,18 +4,19 @@ release_images:
     framework: "tensorflow"
     version: "2.2.0"
     training:
-      device_types: ["cpu", "gpu"]
+      device_types: ["gpu"]
       python_versions: ["py37"]
       os_version: "ubuntu18.04"
-      cuda_version: "cu101"
-      dlc_version: "1.1"
+      cuda_version: "cu102"
+      dlc_version: "1.0"
```

Comment:
Version number will be 2.0.

Reply:
We also need logic for inference as well for TFS.

Reply:
Right, good catch. I thought it was 1.0 because it is the first cu102 docker image, but it's v2.0 because it is another version of an already existing TF 2.2 image. Will make this change.